macosx/Snes9x/S9xPrefsConstants.m

```diff
@@ -25,3 +25,4 @@ SNES9X for Mac OS (c) Copyright John Stiles
 NSString * const kJoypadPlayerPrefs = @"JoypadPlayers";
 NSString * const kShowFPSPref = @"ShowFPS";
 NSString * const kVideoModePref = @"VideoMode";
+NSString * const kMacFrameSkipPref = @"MacFrameSkip";
```

Comment:
One minor nitpick. You might just want the string here to be "FrameSkip". It's user visible if anyone ever looks in the preference plist file. Users don't really care if this is the frame skip variable the Mac port uses to keep track of things as opposed to the variable used in the cross-platform section of the code.